plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/CodeModernizerManager.kt

```diff
@@ -240,7 +240,7 @@ class CodeModernizerManager(private val project: Project) : PersistentStateCompo
         // New projectDetails metric should always be fired whether the project was valid or invalid
         CodetransformTelemetry.projectDetails(
             codeTransformSessionId = CodeTransformTelemetryState.instance.getSessionId(),
-            result = if (!validationResult.valid) Result.Failed else Result.Unknown,
+            result = if (!validationResult.valid) Result.Failed else Result.Succeeded,
```

Comment:
Dhruvi mentioned this cannot be unknown, should be Failed or Succeeded.

Reply:
Interesting, I did not know that.. Its undefined for VSCode which is a valid value. Maybe null would be better here. Its not a true "success metric"

Reply:
Pretty sure `result` has to always be present though